src/App.js

```diff
@@ -1,29 +1,4 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+import Calculator from './components/Calculator';
 
+const App = Calculator;
 export default App;
```

Comment:
- [ ] App.js component should be class-based. Kindly make sure to add a similar below code in App.js.
```
class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

Reply:
Done, thanks!